src/View/Home/index.html.twig

```diff
@@ -120,5 +120,74 @@
     </section>
     <!-- ==================== Fin section Horaires ========================== -->
 
+    <!-- ============== Début section Vie sportive ============= -->
+    <section id="viesportive">
+        <div class="container">
+            <div class="col-lg-12 text-center">
+                <h2>Vie sportive</h2>
+                <div class="h2_separator"></div>
+            </div>
+
+            <div class="row justify-content-around">
+                {% for event in events %}
+                <div class="card rounded-0 col-sm-12 col-lg-3 p-1 card-evt">
+                    <div class="text-center m-2" >
+                    <h5 class="card-title-evt-date">
+                        {% if event.date_begin > event.date_end %}
```

Comment:
Si je ne me trompe pas, la condition est : Si la date de début est supérieure à la date de fin ?! (ca serait plutot l'inverse)
Et donc le else (si la date de début est inférieure ou égale à la date de fin) dans ce cas j'affiche un évènement sur un jour ?
La différence entre deux dates ne valide pas que l'event se passe sur plusieurs jours (à la rigueur l'égalité mais ca dépend de vos règles de gestion) ?

Reply:
ça ne serait pas plutot {% if event.date_end %} tout simplement?

Reply:
Non ok avec @open-it car la date de Début peut etre la date de fin dans notre gestion.
Je modifie